tests/integration/bundle-testing.yaml.j2

```diff
@@ -0,0 +1,55 @@
+---
+bundle: kubernetes
+applications:
+  alertmanager:
+    # charmhub: alertmanager-k8s
+    # local: ./../alertmanager-operator/alertmanager-k8s_ubuntu-20.04-amd64.charm
+    charm: {{ alertmanager|default('alertmanager-k8s') }}
+    scale: 1
+    {%- if alertmanager is defined and alertmanager.endswith('.charm') %}
+    resources:
+        alertmanager-image: "ubuntu/prometheus-alertmanager"
+    {%- else %}
+    channel: edge
+    {%- endif %}
+  prometheus:
+    # charmhub: prometheus-k8s
+    # local: ./../prometheus-operator/prometheus-k8s_ubuntu-20.04-amd64.charm
+    charm: {{ prometheus|default('prometheus-k8s') }}
+    scale: 1
+    {%- if prometheus is defined and prometheus.endswith('.charm') %}
+    resources:
+      prometheus-image: "ubuntu/prometheus:latest"
+    {%- else %}
+    channel: edge
```

Comment:
Eventually the channels may also need to be parameterized.

Reply:
Good point! I think one and same channel for everyone would make sense, i.e. change to 
`channel: {{ channel|default('stable') }}`